app/command_utils.cc.template

```diff
@@ -37,17 +43,26 @@ base::span<const int> GetCommands() {
   return kCommandIds;
 }
 
-base::StringPiece GetCommandName(int command_id) {
+std::string GetCommandName(int command_id) {
   DCHECK(base::FeatureList::IsEnabled(features::kBraveCommands) ||
          commander::CommanderEnabled())
       << "This should only be used when |kBraveCommands| is enabled.";
-  auto it = kCommandInfos.find(command_id);
-  CHECK(it != kCommandInfos.end())
+  auto it = kCommandTranslationIds.find(command_id);
+  CHECK(it != kCommandTranslationIds.end())
       << "Unknown command " << command_id
       << ". This function should only be used for known "
          "commands (i.e. commands in |GetCommands()|). "
          "This command should probably be added.";
-  return it->second.data();
+
+  auto result = l10n_util::GetStringUTF8(it->second);
+  
+  // Note: A bunch of the Chromium strings have a random &amp; in them. I'm not
```

Comment:
Ampersand denotes a shortcut key in a context menu. You can trigger an menu item with the ampersand-ed key. e.g. &New window(On windows, this will look like underlined N), then pressing N in the menu will trigger it.

Reply:
Ah cool - that's real handy to know, thanks @sangwoo108 :smile: